lib/aur-sync

```diff
@@ -321,45 +321,34 @@ fi
 # Link build files in the queue (absolute links)
 parallel -X ln -s "$(pwd -P)"/{} "$tmp_view" :::: "$tmp"/queue
 
-# Explicitly define if exit success is to be trusted (#561)
-trust_exit_success=0
-
-if type -P >/dev/null vifm; then
-    # Avoid directory prefix in printed paths (#452)
-    viewer() ( cd "$1"; vifm -c 'set vifminfo=' -c 'view!' -c '0' - )
-    trust_exit_success=1
-else
-    # shellcheck disable=SC2086
-    viewer() ( cd "$1"; command -- ${PAGER:-less -K -+F} )
-
-    # Trust our fallback
-    if [[ -z $PAGER ]]; then
-        trust_exit_success=1
-    fi
-fi
-
 # Check if dependency graph is valid
 if (( graph )); then
     aur graph "$tmp_view"/*/.SRCINFO >/dev/null
 fi
 
 if (( view )); then
-    # AUR_PAGER exit codes are respected by definition.
+    # Avoid directory prefix in printed paths (#452)
+    viewer() ( cd "$1"; vifm -c 'set vifminfo=' -c 'view!' -c '0' - )
```

Comment:
move this to the `elif type -P vifm >/dev/null; then` branch, perhaps?

Reply:
Good point, done